contracts/cwd-core/src/msg.rs

```diff
@@ -39,18 +38,17 @@ pub struct InstantiateMsg {
     /// Instantiate information for the core contract's voting
     /// power module.
     pub voting_module_instantiate_info: ModuleInstantiateInfo,
-    /// Instantiate information for the core contract's
-    /// proposal modules.
-    // NOTE: the pre-propose-base package depends on it being the case
-    // that the core module instantiates its proposal module.
+    /// Instantiate information for the core contract's proposal modules.
+    /// NOTE: the pre-propose-base package depends on it being the case
+    /// that the core module instantiates its proposal module.
```

Comment:
this is actually an intentional thing where i try and separate "comments for a developer" from "comments for someone interacting with this contract". wdyt? if you find it confusing, then it probably is

Reply:
I think these comments are relevant to show in generated code documentation (e.g. docs.rs), which _is_ for developers. : )